app/scripts/controllers/taskDetails.js

```diff
@@ -57,4 +57,46 @@ angular.module('openhimConsoleApp')
     /**         Task Calculation Functions           **/
     /**************************************************/
 
-  });
\ No newline at end of file
+    function updateTaskWithStatus(task, status) {
+      var updated = new Api.Tasks();
+      updated._id = task._id;
```

Comment:
Yeah, I was trying to avoid that, else it sends through all the fields for the task, like remainingTransactions etc.